examples/multi-project/servers/docker-compose.yml

```diff
@@ -10,7 +10,15 @@ services:
     volumes:
       - waterfall:/server
       - ./waterfall-config:/config
-
+  velocity-proxy:
+    image: itzg/bungeecord
+    environment:
+      TYPE: VELOCITY
+    ports:
+      - "25565:25577"
+    volumes:
+      - velocity:/server
+      - ./velocity-config:/config

```

Comment:
There's now two proxies in this compose project. Please comment out this one and instruct the reader to comment out the waterfall one when using velocity. That way this file remains a working example.